dropwizard-jersey/src/test/java/io/dropwizard/jersey/validation/ValidatingResource.java

```diff
@@ -52,7 +52,7 @@ public String heads(@HeaderParam("cheese") @NotEmpty String secretSauce) {
 
     @GET
     @Path("headCopy")
-    public String heads(@QueryParam("cheese") @NotNull IntParam secretSauce) {
+    public String heads(@QueryParam("cheese") @NotNull @UnwrapValidatedValue(false) IntParam secretSauce) {
```

Comment:
We should really make that change very obvious in the release/upgrade notes for Dropwizard 0.9.0. Otherwise many people won't even realize that their resources are broken.